data/aliases.js

```diff
@@ -505,6 +507,7 @@ exports.BattleAliases = {
 	"bdrill": "Beedrill",
 	"bulu": "Tapu Bulu",
 	"cathy": "Trevenant",
+	"chandy": "Chandelure",
```

Comment:
Uh have you read it being used before? I'm not trying to be rude but that just seems like you're miss typing Chandy after hearing it used